package.json

```diff
@@ -1069,6 +1069,11 @@
                     "type": "integer",
                     "default": 8081,
                     "description": "Port to use when connecting to a CosmosDB Emulator instance"
+                },
+                "azureDatabases.testCosmosAuth": {
```

Comment:
I intend to keep this setting until the testing is done. This allows us to force testing the auth credential and compare with key credentials without going back and forth toggling the disableLocalAuth on the resource.